src/components/Articles/BookmarkArticle.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import { bookmarkArticle } from "../../redux/actioncreators/bookmarkArticleActions";
+import { connect } from 'react-redux';
+import { Button } from 'semantic-ui-react';
+
+export class BookmarkArticle extends Component {
+    handleClick = () => {
+        this.props.bookmarkArticle("starting-up")
+        console.log("--laini--");
+    }
```

Comment:
What does this piece of code do? @asirvex

Reply:
I have removed it. I was using it for testing the api requests